### ISSUE TITLE
 TEXTURE_EXTERNAL_OES target with RGB color format as the render target

### ISSUE THREAD

**Issue by shutaozhenzhen** [KhronosGroup/OpenGL-Registry#624]
In `EXT_YUV_target`, it says:

> To perform the YUV rendering capability in this extension an application
> will attach a texture to the framebuffer object as the color attachment.
> If the texture has a target type of TEXTURE_EXTERNAL_OES with YUV color
> format then the GL driver can use this framebuffer object as the render
> target, TEXTURE_EXTERNAL_OES target with RGB color format are not allowed
> with this extension.

So which extension allow  TEXTURE_EXTERNAL_OES target with RGB(and RGBA) color format as  the render target? I searched the project, can not find infomation.

I'm now work in Mali G52 with `EXT_YUV_target` and `EGL_EXT_image_dma_buf_import.txt`, which allow external texture like RA24, and allow bind to framebuffer, write by glClear or shader. But I can not find which extension allow this. Only `EXT_YUV_target` allow bind TEXTURE_EXTERNAL_OES  to framebuffer, but do not allow RGB.

Is it better to provide another extension to ensure this operation?

**Comment by MarkCallow** [comment]
[OES_EGL_image_external](https://registry.khronos.org/OpenGL/extensions/OES/OES_EGL_image_external.txt) defines the `TEXTURE_EXTERNAL_OES` texture target. Any `EGLImage` can be bound to this target. See the EGL image extensions for which those which let you create an RGB or RGBA EGLImage, e.g. [EGL_KHR_gl_image](https://registry.khronos.org/EGL/extensions/KHR/EGL_KHR_gl_image.txt).

**Comment by shutaozhenzhen** [comment]
> [OES_EGL_image_external](https://registry.khronos.org/OpenGL/extensions/OES/OES_EGL_image_external.txt) defines the `TEXTURE_EXTERNAL_OES` texture target. Any `EGLImage` can be bound to this target. See the EGL image extensions for which those which let you create an RGB or RGBA EGLImage, e.g. [EGL_KHR_gl_image](https://registry.khronos.org/EGL/extensions/KHR/EGL_KHR_gl_image.txt).

yes, but it seems they are defined as sampler. Framebuffer do not contain in them

**Comment by zmike** [comment]
@stonesthrow Maybe you want to comment here?
